assets/quick-add-bulk.js

```diff
@@ -52,6 +57,18 @@ class QuickAddBulk extends HTMLElement {
     });
   }
 
+  resetQuantityInput(id) {
+    const input = document.getElementById(id);
+    input.value = input.getAttribute('value');
+    this.isEnterPressed = false;
+  }
+
+  cleanErrorMessageOnType(event) {
+    event.target.addEventListener('keypress', () => {
```

Comment:
is there an `onChange` on the `input` we can use instead of the `keypress` ?

Reply:
It won't work for us. Because `onChange` would be triggered for `input` only when you submit changes and that's not what we want. We literally want to clean the error message for a browser  as soon as we start typing.